index.html

```diff
@@ -581,14 +581,14 @@ <h3>Talks</h3>
 
   <li class="list-group-item clearfix">
     Lecture for the <a href="https://ongardie.net/static/raft/userstudy/">Raft User Study</a>
-    by <a href="http://www.stanford.edu/~ouster/">John Ousterhout</a>, March 2013:
+    by <a href="https://www.stanford.edu/~ouster/">John Ousterhout</a>, March 2013:
     <img class="img-thumbnail pull-right" src="thumbnails/userstudy2013.jpg" />
     <table class="table-condensed">
       <tr>
         <td>Video (screencast)</td>
         <td>
-          <a href="http://youtu.be/YbZ3zDzDnrw">YouTube</a>
-          <a href="http://raftuserstudy.s3-website-us-west-1.amazonaws.com/raft.mp4">MP4</a>
+          <a href="https://youtu.be/YbZ3zDzDnrw">YouTube</a>
+          <a href="https://raftuserstudy.s3-website-us-west-1.amazonaws.com/raft.mp4">MP4</a>
```

Comment:
This one is under my control, but it doesn't work right now over HTTP. It's also responsible for an AWS bill of about $0.17 per month that has been annoying me for years...

Reply:
Ah, here's an https URL for it
```suggestion
          <a href="https://raftuserstudy.s3-us-west-1.amazonaws.com/raft.mp4">MP4</a>
```